packages/pixeloven/cli-addon-generators/src/templates/app-ssr/contribution.md

```diff
@@ -0,0 +1,122 @@
+# Contribution
```

Comment:
I think this should be up to the owner to define. I'd say let's cut this from here.